.gitignore

```diff
@@ -13,3 +13,5 @@ packages
 # Include when developing application packages.
 pubspec.lock
 .packages
+
+.idea
```

Comment:
Please revert this file change. `.idea` files should be ignored in your global .gitignore file

Reply:
@kevmoo done